code/consoleJSON.js

```diff
@@ -4,110 +4,110 @@ consoleJSON.Util = consoleJSON.Util || {};
-var KEY_SEPARATOR = ".";
+var CONSOLE_STYLE_SPECIFIER = '%c';
+var KEY_ESCAPE_CHAR = '/';
+var KEY_SEPARATOR = '.';
 
 consoleJSON.TYPES = {
-  FILTER : "filter",
-  STYLE : "style",
-  FORMAT : "format"
+  FILTER:'filter',
+  STYLE:'style',
+  FORMAT:'format',
 };
 
 consoleJSON.TARGETS = {
-  KEY : "key",
-  VAL : "val",
-  NUM : "number",
-  STR : "string",
-  BOOL : "boolean",
-  NULL : "null",
-  UNDEF : "undefined",
-  ARR : "array",
-  OBJ : "object",
-  ALL : "all",
-  UNUSED : "unused"
+  KEY:'key',
+  VAL:'val',
+  NUM:'number',
+  STR:'string',
+  BOOL:'boolean',
+  NULL:'null',
+  UNDEF:'undefined',
+  ARR:'array',
+  OBJ:'object',
+  ALL:'all',
+  UNUSED:'unused',
 };
 
 consoleJSON.ATTRS = {
-  REMOVE : "remove",
-  HIGHLIGHT : "highlight",
-  FONT_COLOR : "font_color",
-  FONT_SIZE : "font_size",
-  FONT_STYLE : "font_style",
-  FONT_WEIGHT : "font_weight",
-  FONT_FAMILY : "font_family",
-  LINE_LEN : "line_length",
-  COLLAPSE : "collapse",
-  INDENT_AMT : "indent_amt"
+  REMOVE:'remove',
+  HIGHLIGHT:'highlight',
+  FONT_COLOR:'font_color',
+  FONT_SIZE:'font_size',
+  FONT_STYLE:'font_style',
+  FONT_WEIGHT:'font_weight',
+  FONT_FAMILY:'font_family',
+  LINE_LEN:'line_length',
+  COLLAPSE:'collapse',
+  INDENT_AMT:'indent_amt',
 };
 
 consoleJSON.ATTR_TO_CSS = {};
-consoleJSON.ATTR_TO_CSS[consoleJSON.ATTRS.HIGHLIGHT] = "background";
-consoleJSON.ATTR_TO_CSS[consoleJSON.ATTRS.FONT_COLOR] = "color";
-consoleJSON.ATTR_TO_CSS[consoleJSON.ATTRS.FONT_SIZE] = "font-size";
-consoleJSON.ATTR_TO_CSS[consoleJSON.ATTRS.FONT_STYLE] = "font-style";
-consoleJSON.ATTR_TO_CSS[consoleJSON.ATTRS.FONT_WEIGHT] = "font-weight";
-consoleJSON.ATTR_TO_CSS[consoleJSON.ATTRS.FONT_FAMILY] = "font-family";
+consoleJSON.ATTR_TO_CSS[consoleJSON.ATTRS.HIGHLIGHT] = 'background';
+consoleJSON.ATTR_TO_CSS[consoleJSON.ATTRS.FONT_COLOR] = 'color';
+consoleJSON.ATTR_TO_CSS[consoleJSON.ATTRS.FONT_SIZE] = 'font-size';
+consoleJSON.ATTR_TO_CSS[consoleJSON.ATTRS.FONT_STYLE] = 'font-style';
+consoleJSON.ATTR_TO_CSS[consoleJSON.ATTRS.FONT_WEIGHT] = 'font-weight';
+consoleJSON.ATTR_TO_CSS[consoleJSON.ATTRS.FONT_FAMILY] = 'font-family';
 
 consoleJSON.BEGIN_DELIM = {};
-consoleJSON.BEGIN_DELIM[consoleJSON.TARGETS.ARR] = "[";
-consoleJSON.BEGIN_DELIM[consoleJSON.TARGETS.OBJ] = "{";
+consoleJSON.BEGIN_DELIM[consoleJSON.TARGETS.ARR] = '[';
+consoleJSON.BEGIN_DELIM[consoleJSON.TARGETS.OBJ] = '{';
 
 consoleJSON.END_DELIM = {};
-consoleJSON.END_DELIM[consoleJSON.TARGETS.ARR] = "]";
-consoleJSON.END_DELIM[consoleJSON.TARGETS.OBJ] = "}";
+consoleJSON.END_DELIM[consoleJSON.TARGETS.ARR] = ']';
+consoleJSON.END_DELIM[consoleJSON.TARGETS.OBJ] = '}';
 
 consoleJSON.SEP = {};
-consoleJSON.SEP[consoleJSON.TARGETS.ARR] = ",";
-consoleJSON.SEP[consoleJSON.TARGETS.OBJ] = ",";
+consoleJSON.SEP[consoleJSON.TARGETS.ARR] = ',';
+consoleJSON.SEP[consoleJSON.TARGETS.OBJ] = ',';
 
 consoleJSON.KEY_VAL_SEP = {};
-consoleJSON.KEY_VAL_SEP[consoleJSON.TARGETS.OBJ] = ": ";
```

Comment:
confirmed: the whitespace is unnecessary

ex:
`console.log('%csalami %cprosciutto %cmeatball %cpastrami %cquail','color: blue','font-size: 20px','font-style: italic','font-weight: bold','background: yellow');`
vs.
`console.log('%csalami %cprosciutto %cmeatball %cpastrami %cquail','color: blue','font-size: 20px','font-style: italic','font-weight: bold','background:yellow color:red');`